marconi-chain-index/test/Integration.hs

```diff
@@ -1,261 +0,0 @@
-{-# LANGUAGE DataKinds #-}
```

Comment:
Why was this deleted?

Reply:
It's already in the module Spec.Marconi.ChainIndex.Indexers.ScriptTx, I think it was just forgotten to be deleted.